docs/sts.md

```diff
@@ -24,16 +24,24 @@ The token is a projected ServiceAccount into the Pod, and is short lived for an
 
 ### Steps to install an OpenShift Cluster with STS
 
-1. Set `$RELEASE_IMAGE` to point to a sufficiently new OpenShift release
+1. Set the variable `$RELEASE_IMAGE`
+
+   `$RELEASE_IMAGE` should be a sufficiently new OpenShift release image that you want to deploy in your cluster.
```

Comment:
This is ambiguous. By `sufficiently new` means how new? We have [support matrix](https://github.com/openshift/cloud-credential-operator#support-matrix) for that. It would be great if we can mention the exact version here or some kind of reference to the support matrix